htdocs/core/modules/export/export_csvutf8.modules.php

```diff
@@ -0,0 +1,379 @@
+<?php
+/* Copyright (C) 2006-2013 Laurent Destailleur  <eldy@users.sourceforge.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ *		\file       htdocs/core/modules/export/export_csv.modules.php
+ *		\ingroup    export
+ *		\brief      File of class to build exports with CSV format
+ */
+
+require_once DOL_DOCUMENT_ROOT.'/core/modules/export/modules_export.php';
+
+// avoid timeout for big export
+set_time_limit(0);
+
+/**
+ *	Class to build export files with format CSV
+ */
+class ExportCsvUtf8 extends ModeleExports
```

Comment:
It seems this class is a copy of the class for iso but with just few lines changed.
So why not don't you make an inherit 
 class ExportCsvUtf8 extends ExportCsv
And implement into class only properties or method with change for utf8 ?